paddle/fluid/framework/parallel_executor.cc

```diff
@@ -351,6 +351,8 @@ ParallelExecutor::~ParallelExecutor() {
   for (auto &p : member_->places_) {
     platform::DeviceContextPool::Instance().Get(p)->Wait();
   }
+  member_->nccl_ctxs_.reset(NULL);
```

Comment:
These two is to fix Paddle bug? Please add comment to note this.

Reply:
Fix memory leakage. I will add later.